### ISSUE TITLE
I can't import anything

### ISSUE THREAD

**Issue by ricsam** [pirix-gh/ts-toolbelt#20]
#### Describe the bug
The exports are empty. Does not matter if I use `import { O } ...` or `import tb ...`

![image](https://user-images.githubusercontent.com/986386/61653402-7f3b5480-acba-11e9-9405-744946b0fbcd.png)

happens in both ts-toolbelt@2.1.5 and ts-toolbelt@3.0.0-11

#### Reproduce the bug
```bash
npx create-react-app tb-test --typescript
cd tb-test
yarn add ts-toolbelt
code .
```

#### Possible Solution

![image](https://user-images.githubusercontent.com/986386/61653829-79923e80-acbb-11e9-9ed2-df805bcd2de7.png)
Maybe these should be exported? right now it is `export {};`

the `export declare type` does not seem to be picked up.

Maybe this only happens in CRA. 🤔 

I changed the default tsconfig.json in CRA to the one that is used in this project and the imports still did not work.

**Comment by pirix-gh** [comment]
Try restarting the TS service or your IDE. Does it work?

**Comment by ricsam** [comment]
Nope :(, tested two versions of vscode and restarted the ts server

**Comment by ricsam** [comment]
I have now managed to get it to work somehow. I would say that this is a bug with typescript and not this library. I managed to get it to work by changing the declare export to export and explicitly importing the specific module I wanted to use. Thereafter typescript magically "registered" the existence of the files, so I was able to revert my changes and the exports work as expected. Weird. Alright, I will close this issue for now :)
